tests/expected/function-contract/interior-mutability/transmute/oncecell.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+
+/// The objective of this test is to check the modification of an OnceCell used as interior mutability in an immutable struct
```

Comment:
This test is only using contract for checking, so it also does not require transmute. Please move.

That said, you could add a similar test that instead is written with transmute to enable stubbing. I would highly discourage doing so, since there's no guarantee about the OnceCell layout. If you do write one, please add documentation saying this is just done for test purpose.